lib/puppet/provider/mongodb.rb

```diff
@@ -132,19 +131,19 @@ def self.mongo_eval(cmd, db = 'admin', retries = 10, host = nil)
     cmd = mongorc_file + cmd if mongorc_file
 
     out = nil
-    retry_count.times do |n|
```

Comment:
`n` is still referenced below.  Fixed in https://github.com/voxpupuli/puppet-mongodb/pull/537